src/modules/Layout/Title/Title.scss

```diff
@@ -0,0 +1,8 @@
+@import 'index.scss';
+
+.j-title {
+	font-size: 25px;
```

Comment:
The headers should have been initialized already with standard sizes for each one, same with it's line heights, imo is not a good idea to have a size specific for this h1